news/_posts/core-weekly/2019-05-03-coreweekly-week-17-2019.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 17 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-05-03 15:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 22th to Sunday 28th of April 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+That was amazing to see the huge mobilization to find workarounds in a hurry, during the weekend, to fix the [spam issue](http://build.prestashop.com/news/fighting-against-spamming-again/) with the account creation form.
+
+You will see in this core-weekly the pull requests that fix this issue.  And, in case your missed it, two versions of PrestaShop have been released this morning: [1.6.1.24 and 1.7.5.2](http://build.prestashop.com/news/prestashop-1-7-5-2-1-6-1-24-maintenance-release/). 
```

Comment:
```suggestion
You will see in this core-weekly the pull requests that fix this issue. And, in case your missed it, two versions of PrestaShop have been released this morning: [1.6.1.24 and 1.7.5.2](http://build.prestashop.com/news/prestashop-1-7-5-2-1-6-1-24-maintenance-release/). 
```